src/odemis/gui/util/__init__.py

```diff
@@ -136,7 +136,15 @@ def get_home_folder():
     """ Return the home directory of the user running the Odemis GUI
     """
     # fall-back to HOME
-    folder = os.path.expanduser(u"~")
+    if sys.platform.startswith('linux'):
+        folder = os.path.expanduser(u"~")
+    elif sys.platform.startswith('win32'):
+        # Import functions here because wintypes in ctypes library cannot be opened in linux
+        from odemis.gui.util.knownpaths import get_path, FOLDERID
+        folder = get_path(getattr(FOLDERID, "Profile"))
```

Comment:
You probably can just write: `get_path(FOLDERID.Profile)`.
If so, replace the other calls too.